frontends/pytorch/csrc/builder/graph_importer.cpp

```diff
@@ -0,0 +1,246 @@
+//===- graph_importer.cpp -------------------------------------------------===//
+//
+// This file is licensed under a pytorch-style license
+// See frontends/pytorch/LICENSE for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "graph_importer.h"
+
+#include "mlir-c/Diagnostics.h"
+#include "mlir-c/StandardAttributes.h"
+#include "mlir-c/StandardTypes.h"
+
+namespace py = pybind11;
+using namespace torch_mlir;
+
+GraphImporter::GraphImporter(std::shared_ptr<torch::jit::Graph> graph,
+                             MlirMappingOptions mappingOptions)
+    : graph(std::move(graph)), mappingOptions(std::move(mappingOptions)) {}
+
+std::shared_ptr<GraphImporter> GraphImporter::forPythonJitFunc(
+    torch::jit::Function *function,
+    GraphImporter::MlirMappingOptions mappingOptions) {
+  if (!function->isGraphFunction()) {
```

Comment:
When is it not a graph function? (maybe leave a comment)

Reply:
// Disallow an attempt to compile a native function.

I believe that the `Function` super-type also includes various flavors of native functions that do not have a Graph. There appears to be some cruft in the PyTorch<->Pybind boundary which genericizes these things and I was being cautious.